src/api/tokens/index.ts

```diff
@@ -30,3 +36,28 @@ export const getChainTokens = ctx => {
     ctx.status = 404;
   }
 };
+
+export const getChainToken = ctx => {
+  try {
+    const token = getTokenById(ctx.params.tokenId, ctx.params.chainId);
+    ctx.status = token ? 200 : 404;
+    ctx.body = token ?? {};
+  } catch (err) {
+    console.error(err);
+    ctx.status = 500;
+  }
+};
+
+export const getChainNatives = ctx => {
+  try {
+    const chainId = ctx.params.chainId;
+    const native = getTokenNative(chainId);
+    const wrapped = getTokenWrappedNative(chainId);
+
+    ctx.status = native || wrapped ? 200 : 404;
+    ctx.body = [native, wrapped];
```

Comment:
```suggestion
    ctx.body = { native, wrapped };
```
Can we return this as an object, rather than an array?

Reply:
It'll make it easier for those consuming